lib/src/arithmetic/divider.dart

```diff
@@ -249,13 +252,7 @@ class MultiCycleDivider extends Module {
                   ~bBuf.getRange(0, dataWidth - 2).or() &
                   (signOut ^ signNum),
               then: [
-                // special logic for when a is also most negative #
```

Comment:
Since you are in an If block, would you want to stay with If/Else and not insert a mux?
If you have the unit tests to exercise all paths through this logic, this is a quick refactor.

Reply:
That's fine with me. Since it's quick and non-functional here, I can add it to this PR.